pvlib/test/test_pvsystem.py

```diff
@@ -640,7 +709,51 @@ def test_i_from_v(fixture_i_from_v):
     # Convergence criteria
     atol = 1.e-11
 
-    I = pvsystem.i_from_v(Rsh, Rs, nNsVth, V, I0, IL)
+    I = pvsystem.i_from_v(Rsh, Rs, nNsVth, V, I0, IL, method='lambertw')
+    assert(isinstance(I, type(I_expected)))
+    if isinstance(I, type(np.ndarray)):
+        assert(isinstance(I.dtype, type(I_expected.dtype)))
+        assert(I.shape == I_expected.shape)
+    assert_allclose(I, I_expected, atol=atol)
+
+
+@requires_scipy
+def test_i_from_v_brentq(fixture_i_from_v):
```

Comment:
same simplification as suggested for v_from_i method tests above

Reply:
agree